### ISSUE TITLE
Shut up and take my money! Turns out its not a software bug, it's UI recursive confusion. 

### ISSUE THREAD

**Issue by MaslowsAngels** [twitter/opensource-website#293]
** Thought this was a bug, but turns out it's consultation on a monetization conversion problem, so... whatever, consulting fee waived **

**Describe the fumbled bag**
Paid $8/month or similar, 1 month ago. 
>> on Android/FireFox

Still unable to message anyone. Can't even message @Premium support....
More a marketing problem I finally realize. Can't even give free consulting to @Premium. OK suit urselves X

So I go to premium hub 
>> "Welcome to premium hub" graphics
>> ID verification 
>> ID verification 
>> Press   "Subscribe to premium"     
>> **recursion ** 
>> "Welcome to premium hub" graphics
>> cycle repeats until heat death of universe

>> Try message @Premium on desktop
>> "Get X Premium to message this user"
>> Upgrade  (ok, you want more money???)
>> Looks like you bought this subscription on the X Android app. You’ll need to manage or cancel your subscription from there.

>> KK I figured it out... 

But, X fumbled the bag. Nowhere does it say "pay more money pleb"
At least make it clear that you want to take my money. 
We all assumed it was a "let that sink in" problem.
We all assumed Elon gutted X so much it couldn't even be optimized for profit.

Nope. Just a "Rory Sutherland" marketing problem.

**Comment by Thermotronica** [comment]
yes but if they find out, they still reserve the right to send you a "request" even if youve double paid multiple times for years! in a sense its glad they dont believe in "time" traditionally, but i do see how a few days not being charged automatically a normal human may go," hmm maybe since i paid 20 times this year, theyll forget they are late in requesting for the cash $$$ since they dont even remember what version youre ultimately on "

and then theyll be like, "oh yea, we're on THEIR version, but i need to bill this through legal, and legal is only sure it wants to destroy the universe, and not worried about a silly little thing like numbers, arg only letters for me 🧟‍♂️" 

and then they run it through legal and theyre like "argg can we destroy their life?" and they all nod and forget they can just send a venmo request some like "hey, noticed a glitch, would you like to come and work for us, or you could send the -$543... oh wait no you cant, we actually owe you money uwu 🤪✌️ so quirky" 

so yea anyways, probably true, anyone can "pay more" but its probably a glitch, and i cant send a venmo request through github yet, but cool story!
